### DISCUSSION TITLE
萤门

### DISCUSSION THREAD

**Discussion by Love-Yoimiya** [share121/inter-knot#3034] [其他]
![121418387_p0_master1200](https://github.com/user-attachments/assets/ad33d7ba-c057-4cc3-930e-42e1b781c11e)

**善意不断，萤门不朽。总会有一片夜晚迎来属于它的萤光，在此夜中独行的人会找到他心中之岸，微弱而璀璨的萤光将带领我们走向重逢。
若此间有比存护更坚固之盾，那便是挡在她身前的我们；若此间有比丰饶更永恒之物，那便是我等对她至死不渝的爱意；若此间有比欢愉更快乐之事，那便是我等此刻相聚于此见证她的到来。
再次祝福看到此段的诸位，愿你们终会找到自己的心中眷属，即便是存在于虚幻之中，祂也对你有现实中不可替代的意义和情绪价值。若你已经寻得那束花朵，也请继续握紧祂，将其植入内心之土，终会绽放出花的海洋。
最后请让我以《流萤集》中的部分适配我心绪之美词做结尾:
少女，您的美丽
像尚未成熟的果实，
因打不破的秘密
而心情紧张。
流萤在绿叶丛中
闪闪烁烁，
使繁星为之惊讶。
你刹那间不经意的馈赠，
像秋夜的流星，
在我的身心深处着了火。
我堕入情网，
为你这个人儿
付出了没完没了的相思债。
在宴会结束的时候，
让我在爱的美满结合里，
把我留在人生的花与果里的
一切欢乐统统献给你。**
### 萤门
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Comment by Griffin233** [comment]
萤萤萤！魔萤身犯了<img src="https://bbs-static.miyoushe.com/static/2024/06/19/cd8d307076ea29a6095f34e40dc5b5f8_4644832424539789103.png" alt="流萤 比心" width="75" >
